os.hpp

```diff
@@ -24,33 +24,46 @@
 
 // Define the user struct
 enum userID {sys, u1, u2};
-struct User {
-  userID id;
-  bool running;
+userID nextUser(userID user) {
```

Comment:
This can be modified to an operator overload; this way, when moving to the next user, we can simply use `currentUser++;`